runtime/ts/relation.ts

```diff
@@ -15,14 +15,20 @@ export class Relation extends Entity {
   entities: Entity[];
 
   constructor(...entities) {
-    super();
+    super(undefined);
```

Comment:
undefined is not null, this will probably stop working when we switch on strict checking. Also, semantically userID is optional, so you shouldn't have to specify anything here.

Reply:
fixed